PhiferTristan/readers-client
# Book Form and POST method

Added a BookForm component where a user can fill in the fields for title, author, ISBN number, cover image url, and check which category or categories the book belongs to. Upon clicking the "Submit Book" button, a POST method occurs, creating the book in the database and navigating the user back to the /books view, where they can see their newly submitted book.

### Steps to Test

1. Pull branch "tp-create-book" and switch to it
2. ensure API is connected to localhost
3. npm run dev in client side
4. log into app (credentials are autofilled)
5. navigate to `http://localhost:"your-port-number-here"/createbook` or click the "Create a book" link in the NavBar
6. Fill in the form accordingly and click the "Submit Book" button
7. Scroll down to see if your new book was posted to the databse with the correct information, or check the sqlite3 file in your server-side to check if more information, like the book's id or user_id, is correct
